<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiniTexteditor</class>
 <widget class="QMainWindow" name="MiniTexteditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>text_editor_UI_test</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>701</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QWidget
{
background-color: #323B36;
	background :rgba(0,0,0,0.9);
	
	border: none;
	border-radius: 5px;
	padding: 5px;
color: #fff;

}
</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>561</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	background-color: #323B36;
	background :rgba(0,20,18,184.9);
	border: none;
	border-radius: 8px;
	padding: 5px;
color: #fff;
font-size: 20px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>350</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{

	background-color: #B9A7BB;
background :rgba(0,0,10,100);
	color: #fff;
	font-size: 12px;
	font-weight: bold;
	border: none;
	border-radius: 8px;
	padding: 5px;


}
QPushButton::pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

}
QPushButton::hover
{
		background-color: #6d286e;

}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Undo">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #B9A7BB;
background :rgba(0,0,10,100);
	color: #fff;
	font-size: 10px;
	font-weight: bold;
	border: none;
	border-radius: 8px;
	padding: 5px;

}
QPushButton::pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

}
QPushButton::hover
{
		background-color: #6d286e;

}</string>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Redo">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>410</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #B9A7BB;
background :rgba(0,0,10,100);
	color: #fff;
	font-size: 10px;
	font-weight: bold;
	border: none;
	border-radius: 8px;
	padding: 5px;

}
QPushButton::pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

}
QPushButton::hover
{
	background-color: #6d286e;

}</string>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
QWidget
{
	background-color: #323B36;
	background :rgba(0,20,18,184.9);
	border: none;
	border-radius: 8px;
	padding: 5px;
    color: #fff;
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>300</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #8733ff;
background :rgba(0,0,0,0);
	color: #CCC;
	font-size: 10px;

	font-weight: bold;
	border: none;
	border-radius: 8px;
	padding: 5px;

}
QPushButton::pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

}
QPushButton::hover
{
	color: #fff;

}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #B9A7BB;
background :rgba(0,0,10,100);
	color: #fff;
	font-size: 10px;
	font-weight: bold;
	border: none;
	border-radius: 8px;
	padding: 5px;

}
QPushButton::pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

}
QPushButton::hover
{
	background-color: #6d286e;

}</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Update">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
		background-color: #B9A7BB;
background :rgba(0,0,10,100);
	color: #fff;
	font-size: 10px;
	font-weight: bold;
	border: none;
	border-radius: 8px;
	padding: 5px;

}
QPushButton::pressed
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));

}
QPushButton::hover
{
		background-color: #6d286e;

}</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>-40</y>
      <width>891</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	background-color: #603F8B

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Desktop/New folder/fin21al.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>480</y>
      <width>256</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
QWidget
{
background-color: #323B36;
	background :rgba(0,0,0,0.9);
	
	border: none;
	border-radius: 5px;
	padding: 5px;
color: #fff;

}
</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="lineCount">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>10</y>
      <width>31</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QWidget
{
background-color: #323B36;
	background :rgba(0,0,0,0.7);
	
	border: none;
	border-radius: 5px;
	padding: 5px;
color: #fff;

}
</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>textBrowser</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>Add</zorder>
   <zorder>Undo</zorder>
   <zorder>Redo</zorder>
   <zorder>textEdit</zorder>
   <zorder>clear</zorder>
   <zorder>Delete</zorder>
   <zorder>Update</zorder>
   <zorder>textBrowser_2</zorder>
   <zorder>lineCount</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_project"/>
    <addaction name="actionOpen_project"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionNew_window"/>
    <addaction name="actionClose_window"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSort_line"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="separator"/>
    <addaction name="actionSpell_check"/>
    <addaction name="separator"/>
    <addaction name="actionShow_line"/>
    <addaction name="separator"/>
    <addaction name="actionFind_and_replace"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_project">
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="text">
    <string>Open project</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionNew_window">
   <property name="text">
    <string>New window</string>
   </property>
  </action>
  <action name="actionClose_window">
   <property name="text">
    <string>Close window</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSplit_view">
   <property name="text">
    <string>Split view</string>
   </property>
  </action>
  <action name="actionSort_line">
   <property name="text">
    <string>Sort line</string>
   </property>
  </action>
  <action name="actionUnique">
   <property name="text">
    <string>Unique</string>
   </property>
  </action>
  <action name="actionReverse">
   <property name="text">
    <string>Reverse</string>
   </property>
  </action>
  <action name="actionHide_minimap">
   <property name="text">
    <string>Hide minimap</string>
   </property>
  </action>
  <action name="actionHide_sidebar">
   <property name="text">
    <string>Hide sidebar</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionExpand_Selection">
   <property name="text">
    <string>Expand Selection</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionFind_Replace">
   <property name="text">
    <string>Find &amp; Replace</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionSpell_check">
   <property name="text">
    <string>Spell check</string>
   </property>
  </action>
  <action name="actionShow_line">
   <property name="text">
    <string>Show line</string>
   </property>
  </action>
  <action name="actionFine_Replace">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionFind_and_replace">
   <property name="text">
    <string>Find and replace</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="text_editor_UI_test.qrc"/>
 </resources>
 <connections/>
</ui>
